source/fundamentals/auth.txt

```diff
@@ -49,13 +49,15 @@ relevant mechanism section and select the :guilabel:`Connection String` tab.
 
 The ``MongoCredential`` class contains static factory methods that allow you
 to specify your authentication mechanism and credentials. You can use this
-helper class to keep your code organized. If you choose to use the
-``MongoCredential`` helper class, you need to use the
-``MongoClientSettings.Builder`` class to configure your connection settings
-when constructing your ``MongoClient``. For descriptions of how to use the
-``MongoCredential`` class to specify your authentication method, navigate
-to the relevant mechanism section and select the :guilabel:`MongoCredential`
-tab.
+helper class to separate your logic for retrieving and specifying your
+credentials. This helps you avoid having to parse them into a connection
```

Comment:
Nit: you wouldn't parse them into a connection string.  Rather, you would construct a connection string from the credentials (and other properties)